Meridian59/Data/Models/AdminData/AdminObjectProperty.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    public class AdminObjectProperty : INotifyPropertyChanged
+    {
+        public string PropertyName { get; set; }
+        public string PropertyType { get; set; }
+        
+        private string _propertyValue { get; set; }
```

Comment:
Please avoid using "_" prefixes. Start the property name with a capital letter, and the field name with a non-capital one. Also by default use "protected" rather than "private" - unless you have reasons to, because it fits more cases by default IMO.